testsuite/E71.py

```diff
@@ -64,6 +64,9 @@
 #: E714
 if not X.B is Y:
     pass
+#: E714
+if not X is Y is not Z:
```

Comment:
it's a bit silly but does this still trigger E714 (I think it should, even though it's absurd)?

```python
if not X is not Y:
```

Reply:
Good catch! I've adjusted the regex and added it as a test case.

So it will now catch the pattern `not X is/in` like before except when it's preceded by the keyword `is`.